ui/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Do we need this comment?

Reply:
This was created by Vite automatically, so I'm not entirely sure. I'll look into it.

Reply:
I haven't looked into this, but I will in another PR.